www-client/netsurf/netsurf-3.3.recipe

```diff
@@ -9,65 +9,70 @@ developed, NetSurf is continually evolving and improving."
 HOMEPAGE="http://www.netsurf-browser.org/"
 COPYRIGHT="2003-2014 The NetSurf Browser project"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 # Do NOT use the netsurf-all package. It contains a bunch of subprojects which
 # we build as separate packages so they can easily be used by other projects.
 SOURCE_URI="http://download.netsurf-browser.org/netsurf/releases/source/netsurf-$portVersion-src.tar.gz"
 CHECKSUM_SHA256="23f92a0940f577b9605f2e54786fea3521680d32d6c00f2a3db1eae3f29e3686"
-PATCHES="netsurf-$portVersion.patchset"
+
+if [ $targetArchitecture != x86 ]; then
+	PATCHES="netsurf-$portVersion.patchset"
+else
+	PATCHES="netsurf_x86-$portVersion.patchset"
+fi
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	netsurf$secondaryArchSuffix = $portVersion
+	app:NetSurf$secondaryArchSuffix = $portVersion
```

Comment:
I would omit the $secondaryArchSuffix for the app: entry. If you put it, it allows the gcc2 and gcc4 version of netsurf to be installed at the same time, but they install to the same dir so this is not correct. (do keep it for the other line, however)